packages/client/ui/react-ui/src/providers/CrossmintAuthProvider.tsx

```diff
@@ -65,31 +59,30 @@ export function CrossmintAuthProvider({
     loginMethods = ["email", "google"],
 }: CrossmintAuthProviderProps) {
     const [user, setUser] = useState<SDKExternalUser | undefined>(undefined);
-    const { crossmint, setJwt, setRefreshToken } = useCrossmint(
-        "CrossmintAuthProvider must be used within CrossmintProvider"
-    );
-    const crossmintAuthService = new CrossmintAuthService(crossmint.apiKey);
+    const { crossmint, setJwt } = useCrossmint("CrossmintAuthProvider must be used within CrossmintProvider");
+    // Only create the CrossmintAuth instance once, even in StrictMode, as the constructor calls /refresh
+    // It can only be called once to avoid race conditions
+    const crossmintAuthRef = useRef<CrossmintAuth | null>(null);
+    // biome-ignore lint/correctness/useExhaustiveDependencies: crossmint can't be a dependency because it updates with each jwt change
```

Comment:
hmmm maybe we can drill down on that (in a future PR?)

Reply:
well, how the `Crossmint` object is managed is that a new jwt triggers a new version: https://github.com/Crossmint/crossmint-sdk/blob/main/packages/client/ui/react-ui/src/hooks/useCrossmint.tsx#L23 how would you suggest changing that?

Reply:
I have no clue what that means to React. It feels like this has something to do with component refresh mechanisms related to state change?
I wouldn't know how changing the jwt affects react components, but I do know that we shouldn't need to rebuild the entire service layer because of the jwt changing... it's even expected to change often because of behind-the-scenes refreshs.
Why can't we have a regular memoized value?